src/PyCall.jl

```diff
@@ -59,7 +59,16 @@ typealias PyPtr Ptr{PyObject_struct} # type for PythonObject* in ccall
 #########################################################################
 # Wrapper around Python's C PyObject* type, with hooks to Python reference
 # counting and conversion routines to/from C and Julia types.
+"""
+`PyObject(juliavar)`
```

Comment:
The standard doc style is to indent the first line by four spaces, and not use backticks.  This will make it a code block, not just a portion of the line in code style.  i.e.
```
    PyObject(juliavar)

This converts...
```